config/crd/bases/submariner.io_servicediscoveries.yaml

```diff
@@ -60,6 +60,17 @@ spec:
                 x-kubernetes-list-type: set
               debug:
                 type: boolean
+              deploymentInfo:
```

Comment:
Only reason we put it in status in `Submariner` is that Operator can't udpate a Spec that it listens on. `Submariner` is created outside of operator [helm, user, subctl] so operator can only update Status fields. In case of ServiceDiscovery, it is created by Operator itself. Status in ServiceDiscovery would ideally host information generated by ServiceDiscovery controller.

Reply:
Seems inconsistent to me.

Also this is quite "status" information and not "desired state" so putting it in spec is just conceptually wrong.